### ISSUE TITLE
Organic Debris Field is pretty a pretty boring event

### ISSUE THREAD

**Issue by Imagebard** [d3athrow/vgstation13#9831]
I've been seeing this every other round, too.
What's up with it? Lower the chance a little.

**Comment by Shadowmech88** [comment]
Also #8348 still isn't fixed.

**Comment by Imagebard** [comment]
That's just icing on the cake.

**Comment by Probe1** [comment]
given all these issues it should really be removed.

**Comment by Shadowmech88** [comment]
It was made by @TheEliminator16 right?
Has he been around lately?

**Comment by Probe1** [comment]
Yes but not in a coding capacity.

**Comment by TheEliminator16** [comment]
Yeah, fair enough, this is why I shouldn't code. I don't know what I'm doing.
I suggest that the chance be lowered and all organ bits be removed, just use gibs and the gibbing meteor bodies.
Unless someone else knows how to fix it, because I certainly don't.
